### DISCUSSION TITLE
Database support for Supabase

### DISCUSSION THREAD

**Discussion by codewithkin** [supabase/supabase#13962] [Feature Requests]
Greetings to the wonderful community and the Supabase team. I would like to ask if it is possible to use non-relational databases such as MongoDB with Supabase ? If not.....I would like to humbly request that this feature be added in future releases. Supabase is indeed an amazing application but its convenience would be increased by a fair margin if this feature were to be added

**Comment by pongraczi** [comment]
AFAIK it is not possible, due to that, the backbone of the supabase is postgresql itself.  
Fix me, if I am wrong.

**Reply by codewithkin** [reply]
Guess it's time for me to learn PostGres
